google/cloud/bigtable/internal/table_admin.h

```diff
@@ -258,6 +258,64 @@ class TableAdmin {
         status);
     return result;
   }
+
+  /**
+   * Make an asynchronous request to modify column family of the table.
```

Comment:
s/modify column family/modify the column families of a table/.

Reply:
fixed.